src/group.spec.js

```diff
@@ -22,8 +22,12 @@ describe('group', () => {
   afterEach(async () => {
     await Promise.all(map(
       groupIds,
-      groupId => xo.call('group.delete', {id: groupId})
+      id => xo.call('group.delete', {id})
     ))
+    /*
+      The method "deleteUsers" will be executed only after the resolve
+      of the promise returned by "Promise.all".
+    */
```

Comment:
This comment does not explain anything, this can be understood just by looking at the code, I want to know the *why*.